ci_scripts/check_fauxpas.sh

```diff
@@ -19,4 +19,5 @@ fi
 
 echo "Linting with Faux Pas..."
 
-fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
+set -e fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
```

Comment:
Oh is that true? All of our other scripts redo the set every line so I assumed it only worked for the next command. I can change it.